apps/storefront/src/pages/ForgotPassword/index.tsx

```diff
@@ -49,14 +49,14 @@ export function ForgotPassword({
   });
 
   const navigate = useNavigate();
-  const emailAddressReset = watch('emailAddress');
+  const emailReset = watch('email');
```

Comment:
this keeps making me think it'll be a boolean, e.g. "yes, email was reset"

`newEmail`..?

Reply:
turns out this was only "needed" for captcha, and captcha didn't need it 🤷